contracts/migrations/9_transfer_ownership_and_tokens_for_mainnet.js

```diff
@@ -0,0 +1,47 @@
+const OriginToken = artifacts.require('./token/OriginToken.sol')
+const V00_Marketplace = artifacts.require('./V00_Marketplace.sol')
+
+const tokenMultiSig = '0xe011fa2a6df98c69383457d87a056ed0103aa352'
```

Comment:
Not needed in this PR but maybe add a TODO for us to put all these addresses in a common "addresses.js" file so that they can then be imported by all the migrations scripts. Otherwise I'm worried in the future we may update an address in one migration script but forget to update in the others ?

Reply:
added TODO